packages/one-app-runner/bin/one-app-runner-test.js

```diff
@@ -16,6 +16,23 @@
 
 require('dotenv').config();
 const { spawn } = require('child_process');
+const { waitForOK } = require('../utils/waitForOK');
 
 const command = `one-app-runner --output-file=one-app-test.log --create-docker-network --docker-network-to-join=${process.env.NETWORK_NAME} --use-host`;
-spawn(command, { shell: true, stdio: 'ignore', detached: true }).unref();
+
+const run = async () => {
+  spawn(command, { shell: true, stdio: 'ignore', detached: true }).unref();
+  const port = process.env.HTTP_PORT;
+  const timeout = 200000;
+
+  console.log(`Waiting for one app to start on port ${port}`);
+
+  const val = await waitForOK({
```

Comment:
not a blocker but it would be nice to have a better name for `val`, doesn't tell me much

Reply:
Agreed. I have changed to `resolvedStatus`.